07LessonSeven/homework/index.html

```diff
@@ -1,10 +1,41 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
-<body>
 
+<body>
+    <header>
+        <h1>Profession Portal</h1>
+    </header>
```

Comment:
this is looking very good! the only issue I have is that the heading, nav link, and green box don't exactly line up vertically. Try to make that space from the left the same amount.